numba/tests/test_target_extension.py

```diff
@@ -682,6 +667,76 @@ def dpu_foo_cpu():
         for msg in msgs:
             self.assertIn(msg, str(raises.exception))
 
+    def test_overload_allocation(self):
+        def cast_integer(context, builder, val, fromty, toty):
+            # XXX Shouldn't require this.
```

Comment:
Noted. I guess this is due to the casts not being installed in the dpu target and the inheritance model etc not handling this as yet?

Reply:
yes, and i'm too lazily to put in the cast